src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-list.js

```diff
@@ -0,0 +1,84 @@
+
+module.exports = {
+  tags: ['ss', 'clients', 'clientslist'],
+  'Security server clients list': browser => {
+    const frontPage = browser.page.ssFrontPage();
+    const mainPage = browser.page.ssMainPage();
+    const clientsTab = mainPage.section.clientsTab
+
+    // Open SUT and check that page is loaded
+    frontPage.navigate();
+    browser.waitForElementVisible('//*[@id="app"]');
+
+    // Enter valid credentials
+    frontPage
+      .clearUsername()
+      .clearPassword()
+      .enterUsername(browser.globals.login_usr)
+      .enterPassword(browser.globals.login_pwd)
+      .signin();
+
+    // Switch to clients tab and verify
+    mainPage.openClientsTab();
+    browser.waitForElementVisible(clientsTab);
+
+    // Verify default sorting and list items
+    browser
+      .waitForElementVisible('//th[span[contains(text(),"Name")] and contains(@class, "active") and contains(@aria-sort, "ascending")]')
+      .waitForElementVisible('(//tbody/tr)[1]//span[contains(text(),"TestCom")]')
```

Comment:
Would it make it easier to create a function that generates these nearly identical strings. Something like this:

```
const generateSSClientTableRow = (index, text) => {
  return `(//tbody/tr)[${index}]//span[contains(text(), "${text}")]`
}
```

and then call it:
`generateSSClientTableRow(1, "TestCom")`